.github/workflows/pipeline-deploy-app-test.yaml

```diff
@@ -24,10 +24,10 @@ jobs:
   deploy-firewall:
     uses: NHSDigital/uec-dos-management/.github/workflows/deploy-infrastructure.yaml@latest
     with:
-      environment: dev
+      environment: test
```

Comment:
why has this changed to test from dev?

Reply:
Because this workflow is deploying to the test account. In this context environment = account. What was happening before was the WAF was being built in the dev account and the later job was trying to build  the cloudfront distribution in the test account - and failing because it couldn't find the WAF in the test account